manifests/crane.pp

```diff
@@ -27,6 +27,7 @@
   Stdlib::Absolutepath $ca_cert,
 
   Boolean $debug = false,
+  Stdlib::Fqdn $server_name = $::fqdn,
```

Comment:
Kafo has had issues with `$facts['fqdn']` so here I'm intentionally using the older version